h2o-webserver-iface/src/main/java/water/webserver/iface/H2OHttpConfig.java

```diff
@@ -28,4 +30,54 @@
 
   public String context_path;
 
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) return true;
```

Comment:
I'm sure that you thought about that, but is there any specific reason to implement the equals method, can't we use the default implementation?

Reply:
Default implementation? Do you mean the equals from Object? That is just comparing the object reference.

Or do you mean equals generated at runtime by a utility function/library?

This code is generated by IntelliJ, not handwritten.

Reply:
Cool! What do you think about mentioning that this was generated by IDE?

Reply:
Sure, I will add!